From 3e79a35656461e7143134e0b65b2e78d30df7785 Mon Sep 17 00:00:00 2001
From: Daniel Ebert <daniel.ebert@protonmail.com>
Date: Tue, 19 Sep 2023 16:29:52 +0200
Subject: [PATCH 126/188] Align arguments in a function call in C. Since the
 tree-sitter grammar is not very good at parsing function calls while they're
 being written, this is not yet super useful. However, it prevents the new
 `hybrid` indent heuristic from choosing these lines as a baseline, making it
 more robust.

---
 runtime/queries/c/indents.scm | 3 +++
 1 file changed, 3 insertions(+)

--- a/runtime/queries/c/indents.scm
+++ b/runtime/queries/c/indents.scm
@@ -36,3 +36,6 @@
 (parameter_list
   . (parameter_declaration) @anchor
   (#set! "scope" "tail")) @align
+(argument_list
+  . (_) @anchor
+  (#set! "scope" "tail")) @align
